core/build.gradle.kts

```diff
@@ -73,40 +81,28 @@ kotlin {
     sourceSets {
         commonMain {
             dependencies {
-                api("org.jetbrains.kotlin:kotlin-stdlib-common")
+                api("org.jetbrains.kotlin:kotlin-stdlib")
```

Comment:
This is not correct until stdlib becomes a mpp library. Better not to use stdlib dependency at all and let KGP configure it automatically.

Reply:
done